### PR TITLE
Reverting ACL changes from create-serverless-service and updating ACL request in service-onboarding file.

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* ACL implementation in create-service work flow

### Description of the Change

ACL is updated for all 3 categories once the repo is created.

### CODE REVIEW

File: builds/jazz-build-module/acl-module.groovy [hunk 1]
```diff
@@ -4,12 +4,15 @@ import groovy.json.JsonOutput
 
 def updateServiceACL(serviceId, auth_token, aclUrl) {
 	try {
-		def categoryList = ['code', 'deploy']
+		def categoryList = ['manage','code', 'deploy']
 		def policiesList = []
+
 		for (category in categoryList) {
+			let permission = 'write';
```
Comment (SiniWilson) [inline builds/jazz-build-module/acl-module.groovy]: let or def ?

File: builds/jazz-build-module/acl-module.groovy [hunk 2]
```diff
@@ -4,12 +4,15 @@ import groovy.json.JsonOutput
...
+			if(category === 'manage') permission = 'admin'
```
Comment (SiniWilson) [inline builds/jazz-build-module/acl-module.groovy]: I think "=="

File: builds/jazz-build-module/acl-module.groovy (hunk 1 above)
Comment (supritat) [reply builds/jazz-build-module/acl-module.groovy]: Updated.

File: builds/jazz-build-module/acl-module.groovy (hunk 2 above)
Comment (supritat) [reply builds/jazz-build-module/acl-module.groovy]: updated

**Pull request merged**